testdata/orig_with_vis_config_and_transform.json

```diff
@@ -0,0 +1,12 @@
+{
+  "kind": "qri:ds:0",
```

Comment:
keep in mind `kind` is now spelled `qri`, with no `qri:` prefix on the value 😉

Reply:
added issue #4 to update this